cloudfoundry.md

```diff
@@ -130,6 +130,8 @@ cf set-env foia DATABASE_URL [value]
 cf set-env foia FOIA_ANALYTICS_ID [value]
 cf set-env foia FOIA_SECRET_SESSION_KEY [value]
 cf set-env foia DJANGO_SETTINGS_MODULE foia_hub.settings.dev
+cf set-env foia NEW_RELIC_LICENSE_KEY=[value]
+cf set-env foia NEW_RELIC_APP_NAME=[value]
```

Comment:
I think I wasn't completely clear earlier. We're flipping the new relic app name to foia-a, foia-b. But there's no distinction effectively since both of those are just a production instance. That is, we should collect statistics on them separately. I'm assuming if it's just called foia - the history, etc. works out more sensibly when we look at all our New Relic charts and graphs? Does that make sense? If not I'm happy to explain/listen in person.